src/main/java/io/fabric8/jenkins/openshiftsync/BuildConfigWatcher.java

```diff
@@ -45,6 +54,20 @@ public void onClose(KubernetesClientException e) {
     }
   }
 
+  @SuppressFBWarnings("NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE")
+  public void onInitialBuildConfigs(BuildConfigList buildConfigs) {
```

Comment:
Do we need a mutex to ensure that this & any other on* methods aren't run concurrently? Otherwise we could race to update the same job from build config?